app/webpacker/channels/session_channel.js

```diff
@@ -1,8 +1,9 @@
 import consumer from './consumer'
-import CableReady from 'cable_ready'
 
 consumer.subscriptions.create("SessionChannel", {
   received(data) {
-    if (data.cableReady) CableReady.perform(data.operations)
+    if (!data.selector) return;
+
+    document.querySelector(data.selector).innerHTML = data.html;
   }
```

Comment:
I don't think there's a reason to replace CableReady with ActionCable. CableReady is adding a feature, meaning we can use less code.
Remember that the original issue was simply to replace mrujs, not to replace CableReady

`±` Can you please remove/revert this, and the similar commits? 
* Replace bulk invoice CableReady broadcast with ActionCable
* Replace admin report CableReady broadcast with ActionCable

Reply:
Ok, I see that @mkllnk mentioned ActionCable [here](https://github.com/openfoodfoundation/openfoodnetwork/issues/12794#issuecomment-2370000245). I'm not sure if this is what he intended though.

Maikel, do you have an opinion if this change is a good idea or not? There's a couple more commits following which give more context.